package.json

```
@@ -44,7 +43,18 @@
         "command": "crystal.ameba.disable",
         "title": "Crystal Ameba: disable lints (workspace)"
       }
-    ]
+    ],
+    "configuration": {
+      "type": "object",
+      "title": "Crystal Ameba configuration",
+      "properties": {
+        "crystal-ameba.onType": {
+          "type": "boolean",
+          "default": true,
+          "description": "Run Ameba linter while typing (requires ameba > v1.6.1)"
```

I'm not sure that's needed.

```suggestion
          "description": "Run Ameba linter while typing"
```